java/memory/memory-core/src/test/java/org/apache/arrow/memory/util/TestArrowBufPointer.java

```diff
@@ -111,6 +111,7 @@ public void testNullPointersHashCode() {
   }
 
   @Test
+  @SuppressWarnings("ReturnValueIgnored")
```

Comment:
Maybe instead of suppresing we can check that the hash code changes/doesn't change?

Reply:
Changed

Reply:
We can remove the suppression now right?